### ISSUE TITLE
Update EF Core to v7.0.10

### ISSUE THREAD

**Issue by vanlueckn** [koenbeuk/EntityFrameworkCore.Projectables#82]
Currently Projectables is using v7.0.9. It would be nice if it would support v7.0.10.

**Comment by koenbeuk** [comment]
This library is tied to dotnet 6, it should be compatible with v7.0.10. Are you saying that its not?

**Comment by vanlueckn** [comment]
No, I just can't change my EF dependencies to `v7.0.10`. I always get the message that the packages conflict with EntityFrameworkCore.Projectables because this package references `v7.0.9`.

**Comment by koenbeuk** [comment]
This package has a dependency on EFCoreVersion 6.0.0 and is compatible with the latest EF Core 7 release.

**Comment by leoerlandsson** [comment]
We used EntityFrameworkCore.Projectables with both EF v7.0.10 and 7.0.11. It works well.
